app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,19 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+
+    def __init__(self,

```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one